docs/source/nlp/tutorials/quickstarts/plot_token_classification.py

```diff
@@ -0,0 +1,180 @@
+#   include_long_calculation_properties=True, device=device
+# )
+# test.calculate_builtin_properties(
+#   include_long_calculation_properties=True, device=device
+# )
+
+# %%
+# In this example though we'll use pre-calculated properties:
+
+train_properties, test_properties = scierc_ner.load_properties()
+
+train.set_properties(train_properties, categorical_properties=['Language'])
+test.set_properties(test_properties, categorical_properties=['Language'])
+
+train.properties.head(2)
+
+# %%
+# Running Checks
+# ==============
+#
+# Train Test Performance
+# ----------------------
+#
+# Once the :ref:`TextData <nlp__textdata_object>` object is ready, we can run the checks. We'll start by running
+# the :ref:`TrainTestPerformance <nlp__train_test_performance>` check, which compares the performance of the model on
+# the train and test sets. For this check, we'll need to pass the model's predictions on the train and test sets, also
+# provided in the format of an IOB annotation per token in the tokenized text.
+#
+# We'll also define a condition for the check with the default threshold value.You can learn more about customizing
+# checks and conditions, as well as defining suites of checks in our
+# :ref:`Customizations Guide <general__customizations>`
+
+train_preds, test_preds = scierc_ner.load_precalculated_predictions()
+
+from deepchecks.nlp.checks import TrainTestPerformance
+check = TrainTestPerformance().add_condition_train_test_relative_degradation_less_than()
+result = check.run(train, test, train_predictions=train_preds, test_predictions=test_preds)
+result
+
+# %%
+# We can see that the model performs better on the train set than on the test set, which is expected. We can also note
+# specifically that the recall for class "OtherScientificTerm" has declined significantly on the test set, which is
+# something we might want to investigate further.
+#
+# Embeddings Drift
+# ----------------
+#
+# The :ref:`EmbeddingsDrift <nlp__embeddings_drift>` check compares the embeddings of the train and test sets. In
+# order to run this check you must have text embeddings loaded to
+# both datasets. You can read more about using embeddings in deepchecks NLP in our
+# :ref:`Embeddings Guide <nlp__embeddings_guide>`. In this example, we have the embeddings already pre-calculated:
+
+train_embeddings, test_embeddings = scierc_ner.load_embeddings()
+
+
+train.set_embeddings(train_embeddings)
+test.set_embeddings(test_embeddings)
+
+# %%
+# You can also calculate the embeddings using deepchecks, either using an
+# open-source sentence-transformer or using Open AI’s embedding API.
+
+# train.calculate_builtin_embeddings()
+# test.calculate_builtin_embeddings()
+
+# %%
+
+from deepchecks.nlp.checks import TextEmbeddingsDrift
+
+check = TextEmbeddingsDrift()
+res = check.run(train, test)
+res.show()
+
+# %%
+# The check shows the samples from the train and test datasets as points in the 2-dimensional reduced embedding
```

Comment:
do we want to add a link to it (the check)?
(cause as a reader I wonder "how is the dimension reduced" etc, and don't think it's relevant here but would want to easily read further

Reply:
There's one above